_extras/guide.md

```diff
@@ -162,15 +162,15 @@ Tip: use `.head()` method throughout this lesson to keep your display neater for
 
 ### Selection Challenges
 
-* What happens when you type:
+* What happens when you execute:
 
-	`surveys_df[0:3]`
-	`surveys_df[:5]` slicing from first element makes 0 redundant
+	`surveys_df[0:1]` slicing only the first element
```

Comment:
Please keep the first example `[0:3]` as this is a general example of slicing.
Make `[0:1]` the second exampe.

Reply:
I also prefer having a more expansive slice as the first one, especially for new learners who are still adjusting to exclusive slicing.